apps/web/components/Templates/Header.tsx

```diff
@@ -0,0 +1,48 @@
+import Link from "next/link";
+import { useState } from "react";
+import { CloseIcon, HamburgerIcon } from "../../assets/icons";
+import TemplatesNavLinks from "./TemplatesNavLinks";
+
+export default function Header() {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  return (
+    <header className="h-[88px] md:bg-zinc-800">
+      <div className="container py-4">
+        {/* Change height of the div below to `h-96` after filters are enabled */}
+        <div
+          className={`relative z-10 -mx-4 overflow-hidden rounded-lg bg-zinc-700/50 backdrop-blur-lg transition-all duration-300 md:flex md:items-center md:justify-between md:bg-transparent md:p-4 lg:overflow-hidden ${
```

Comment:
Try "clsx" library for this, it's more readable than using conditions in string literals